src/main/java/school/lemon/changerequest/java/generics/pair/PairUtils.java

```diff
@@ -0,0 +1,45 @@
+package school.lemon.changerequest.java.generics.pair;
+
+public class PairUtils {
+
+    public static boolean equals(Pair<?, ?> first, Pair<?, ?> second) {
+        return first.getKey().equals(second.getKey()) && first.getValue().equals(second.getValue());
```

Comment:
What about `NullPointerException` checks? I'll for sure pass `null` as an argument for `first` or `second`.